### DISCUSSION TITLE
Broken hyperlinks in draw-io diagrams

### DISCUSSION THREAD

**Discussion by arnerb** [requarks/wiki#3304] [Error / Bug Report]
# Description
## Hyperlinks added in draw-io diagrams do not work as expected

More precisely
| | link the text within the object | link the object |
|---|---|---|
|1 link per diagram | OK | entire diagram turns into hyperlink |
|2 or more links per diagram | only one of them works | entire diagram turns into hyperlink |

## To Reproduce
1. Open diagram editor
2. add two objects
3. add hyperlinks with "Shift + ALT + L" or click link button:
![add hyperlink button](https://user-images.githubusercontent.com/58850630/104193540-84491200-5420-11eb-89d6-85160d4b6b2a.png)
5. Save and exit, save page
6. Experience broken or bad links

## Expected behavior
The diagrams should work as they do if you open them in a browser.
* [Object texts](https://viewer.diagrams.net/?highlight=0000ff&edit=_blank&layers=1&nav=1&title=#RzZRRT4MwEMc%2FDa8GqBJfHdswUbOHxfjc0FvbWDjsymB%2BetvRDshmYmJifCC0v7te2%2F%2F%2FICJ51ReaNuIFGagojVkfkWWUpmmcpfblyHEgCUmSgXAtmWcj2MpP8DD2tJUM9rNEg6iMbOawxLqG0swY1Rq7edoO1XzXhnK4ANuSqkv6JpkRA72%2Fi0f%2BCJKLsHMS%2B0hFQ7IHe0EZdhNEVhHJNaIZRlWfg3LqBV2GdetvoueDaajNTxZ4Jw5Utf5uUZopu3RBLRYadieafbTuRAthjJP4wRVM1xyRK7gpsRoz7Ii7d7HZFM%2BrUMwm0xDxFzfHoKY9kTXOThadkAa2DS1dpLPNc9qyUnaWuCPtm8HNneyBnUtNb%2BsFOIA20E%2BQv30BWIHRR5sSujH0lO9FEpzpRmOTzDMxNTVA6puJn2uPetuBl%2Fy6%2FOQ38rO2fHcPx%2BsWLF%2Fzp%2F9vwG38hwbY6fhtnWKTXxRZfQE%3D)
* [Objects](https://viewer.diagrams.net/?target=blank&highlight=0000ff&edit=_blank&layers=1&nav=1&title=#RzZRNb8MgDIZ%2FTa5VAlu165q2mbRNPVTVzjRxAysJEaEj3a8fFMiHuva6HaLAg43t104inFZdJklD30UBPEJx0UV4GSGE4jkyL0vOjiQ4SRwpJSs8G8CWfYOHsacnVkA7MVRCcMWaKcxFXUOuJoxIKfTU7CD4NGpDSrgC25zwa%2FrBCkUdfXqMB%2F4CrKQhchL7k4oEYw9aSgqhRwivIpxKIZRbVV0K3KoXdHF%2B6xunfWISavWLw64Fudl%2FWk1QzMneNOZilG022dvKOXNWHx2lSllBn607WmutZ6UQJYdZLqoLQ6PQqM%2B%2Bz6pV56CYAaY5ZrPQlCnYNiS3J9oMiGFUVTaRxCxJ27iOHVgH5urFdUW%2ByC%2BQCroR8sEzEBUoeTYmYeLC3Ph5w0F9PTQvmXtGx40LkPiBKfu7B03NwlcctiOV76q%2B3KWv9zUvTvnRPqVwmo8Fx%2F9X8If4bwU3aPiELuajPxFe%2FQA%3D)

### Motivation / use case:  **Visual wiki** 
It would be especially helpful for displaying process diagrams, in which each process step contains a link to a page/section with more detailed description 

## Host Info
 - OS: [Docker Container (Linux), Ubuntu 18]
 - Wiki.js version: [2.5.170]
 - Database engine: [SQLite 3.31.1 (demo database)]

## Related
https://requarks.canny.io/wiki/p/svg-image-links-are-stripped-down-when-used-in-markdown-pages

**Comment by profzoidberg** [comment]
Hi, 
we have the same problem. Is there a solution? 
regards
